common/config.js

```diff
@@ -19,7 +19,6 @@ function get(name, fallback, options = {}) {
 }
 
 module.exports = {
-  sessionSecret: get('SESSION_SECRET', 'app-insecure-default-session', { requireInProduction: true }),
   apis: {
     oauth2: {
```

Comment:
Could I get you to rip out oauth2 and offenderAssessment whilst you are there, please? 😬

Reply:
done